### ISSUE TITLE
Audio tag events do not fire

### ISSUE THREAD

**Issue by erichulburd** [DioxusLabs/dioxus#541]
**Problem**

`onended`, `ontimeupdate`, `onloadedmetadata` `<audio />` events never fire.

**Steps To Reproduce**

See [main.rs](https://github.com/erichulburd/dioxus-audio-tag-event-bugs/blob/master/src/main.rs) in this simplified app that reproduces the bug: https://github.com/erichulburd/dioxus-audio-tag-event-bugs.

<details>
<summary>tldr</summary>

```rs
fn app(cx: Scope) -> Element {
    let mut metadata = use_state(&cx, || 0);
    let mut timeupdate = use_state(&cx, || 0);
    let mut ended = use_state(&cx, || 0);

    cx.render(rsx!(
        h1 { "Audio Event Bugs" }
        table {
            thead {
                tr {
                    th { "Event" }
                    th { "Count" }
                }
            }
            tbody {
                tr {
                    td {
                        "onloadedmetadata"
                    }
                    td { "{metadata}" }
                }
                tr {
                    td {
                        "ontimeupdate"
                    }
                    td { "{timeupdate}" }
                }
                tr {
                    td {
                        "onended"
                    }
                    td { "{ended}" }
                }
            }
        }
        audio {
            src: "/static/sample.mp3",
            preload: "metadata",
            controls: "true",
            onloadedmetadata: move |_| metadata += 1,
            ontimeupdate: move |_| timeupdate += 1,
            onended: move |_| ended += 1,
        }
    ))
}
```

</details>

**Expected behavior**

When the audio tag loads metadata, updates time, or ends, we want the events to fire. In the sample app, this would show up in the HTML table that contains counts of each event.

**Screenshots**

-

**Environment:**
 - Dioxus version: 0.2.4
 - Rust version:

```shell
% rustc --version
rustc 1.62.1 (e092d0b6b 2022-07-16)
% cargo --version
cargo 1.62.1 (a748cf5a3 2022-06-08)
% rustup --version
rustup 1.25.1 (bb60b1e89 2022-07-12)
```

 - OS info: Apple M1 Max, macOS Monterey v12.3
 - App platform: `web`

**Questionnaire**

- [X] I'm interested in fixing this myself but don't know where to start
- [ ] I would like to fix and I have a solution
- [ ] I don't have time to fix this right now, but maybe later

**Comment by rMazeiks** [comment]
I suspect this is because the events do not bubble.

A similar issue (#248) has been fixed in #452. Though it looks like the events you mention were not addressed by that PR.

**Comment by erichulburd** [comment]
I saw #248 and the fix. What's the source of truth for the event bubble mapping in [pub fn event_bubbles](https://github.com/DioxusLabs/dioxus/pull/452/files#diff-275a6898411b8826af228ce24c8189624177929dfadfcbc917fb5a227bbc0eeeR1328). MDN doesn't indicate which events bubble. The best I've found is a few very old threads that discuss the lack of bubbling on media events.

* https://stackoverflow.com/a/12630384
* https://teamtreehouse.com/community/how-to-handle-event-delegation-with-media-events


I'm wondering if the issue here is that the media events in question are marked as `bubbling: true`, but in fact do not bubble.
